dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/Serializer.java

```diff
@@ -69,11 +72,17 @@ static DimensionList fromLabels(Labels labels) {
 
         if (isDelta) {
           builder.setLongCounterValueDelta(point.getValue());
+          lines.add(builder.serialize());
         } else {
-          builder.setLongCounterValueTotal(point.getValue());
+          Long delta = deltaConverter.convertLongTotalToDelta(metric.getName(), point);
+          if (delta != null) {
+            builder.setLongCounterValueDelta(delta);
+            lines.add(builder.serialize());
+          } else {
+            logger.finest(
+                "Skipping delta line creation, since the value was not present in the cache before this value.");
```

Comment:
```suggestion
                String.format("Skipping delta conversion for metric '%s' since no previous value was present in the cache.", metric.getName()));
```
It would be helpful to print the metric name here is well. With dimensions it could get quite long so probably leaving them out is better.